int64 timestamp
float64 heading
float64 pitch
float64 roll
float64 Ve
float64 Vn
float64 Vu